src/modules/partition/jobs/CreatePartitionTableJob.cpp

```diff
@@ -87,22 +87,34 @@ CreatePartitionTableJob::exec()
     }
 
     QScopedPointer< PartitionTable > table( createTable() );
-    cDebug() << "Creating new partition table of type" << table->typeName()
-             << ", uncommitted yet:\n" << table;
+    cDebug() << "Creating new partition table of type:" << table->typeName()
+             << " - Uncommitted yet:" << table;
```

Comment:
Since @stikonas is doing all kinds of things with dropping Calamares' partition code and using KPMCore instead, I'd hesitate to be changing debugging in that module **right now**. In general, though, this is a useful little bit of polish.

Reply:
Well, right now I don't plan to touch CreatePartitionTableJob. Partially because it is not necessary for LVM (it doesn't have partition table, it is enough to just disable creation of partition table).

On the other hand maybe it will still be a good idea to move to kpmcore code in the future for CreatePartitionTable too... Less code dublication.